development/tools/cmd/certbotauthenticator/README.md

```diff
@@ -0,0 +1,29 @@
+#certbotauthenticator
+
+##Overview
+
+Certbotauthenticator is a binary called by certbot during certificate generation process. It's used in phase of manual DNS challenge authentication. Cerbot in manual mode is passing domain name and authentication token as environment variables to the certbotauthenticator, for the purpose os creating TXT record in a domain, to let LetsEncrypt system validate domain ownership. After validation, certbotauthenticator is called again to clean TXT records created for validation.
+
+##Justification
+
+We made this tool, becasue certbot dns-google plugin doesn't support creating TXT records within DNS Zone existing in different project, than service account used to authenticate in GCP.
```

Comment:
```suggestion
## Justification

We have created this tool because the certbot dns-google plugin does not support creating TXT records within a DNS zone existing in different project than the service account used to authenticate in GCP.
```